feature/home/presentation/home-ui/src/main/java/com/chocolatecake/ui/episode_details/EpisodeDetailsFragment.kt

```diff
@@ -47,12 +47,22 @@ class EpisodeDetailsFragment :
     private fun checkIsLoggedInOrNot() {
         val isLoggedIn = viewModel.state.value.isLoggedIn
         if (isLoggedIn) {
-            showBottomSheet()
+            showBottomSheet(viewModel.state.value.voteAverage)
         } else {
             showSnackBar("You are not logged in \uD83D\uDE22, please log in to rate this episode");
         }
     }
 
+    private fun showBottomSheet(voteAverage: Float) {
+        val bottomSheet = EpisodeRateBottomSheet()
+        bottomSheet.arguments = Bundle().apply {
+            putFloat("voteAverage", voteAverage)
+        }
```

Comment:
try to access the voteAverage from the bottom sheet using the viewModel instead of passing it throught bundle